### ISSUE TITLE
does gblinear have eta parameter?

### ISSUE THREAD

**Issue by junfenglx** [dmlc/xgboost#1020]
I use Python package.

the code is:
```python
num_round= 2900

param = { 'booster': 'gblinear', 'lambda': 0.5, 'alpha': 0.5, 'nthread':5,
         # 'eta': 0.3
        }

dtrain=xgb.DMatrix(X_train,label=y_train)
dtest=xgb.DMatrix(X_test)

watchlist  = [(dtrain,'train')]

start_time = time.time()
bst = xgb.train(param, dtrain, num_round, watchlist)
```

When I don't specify eta prameter, I get big values -> inf -> nan of train-rmse.
When I specify eta prameter, the train error becomes normal.

another related issue is #950

Is there is eta(learning rate) for gblinear booster?
is it's default value is 0.3?
If so, I thing it's better explicitly doc this behavior.
Currently the doc says eta parameter is for tree booster.

Thanks

By the way. another issue is the nthread parameter sometimes doesn't work.
XGBoost use all cores even specify nthread when use gblinear booster.
Should I explicitly set OMP_NUM_THREADS variable?

Thanks again

**Comment by junfenglx** [comment]
closed for no response.

**Comment by tqchen** [comment]
There is indeed learning rate in gblinear. But usually there is no need to set this parameter as long as the result converge. Smaller learning rate indeed help convert when you have skewed dataset.

**Comment by junfenglx** [comment]
Thanks for reply, I get it.
